cpp/client.cc

```diff
@@ -56,18 +56,25 @@ class WalletClient {
     ClientContext context;
     context.set_wait_for_ready(true);
     if (user == "Alice") {
-      context.AddMetadata("authorization", "2bd806c9");
+      std::string token = "2bd806c9";
+      context.AddMetadata("authorization", token);
       context.AddMetadata("membership", "premium");
+      if (affinity) {
+        // use something unique per user as session id.
+        context.AddMetadata("session_id", token);
```

Comment:
Are we supposed to use the same value for session_id and authorization token?

Reply:
I just picked token as it is already per user; but we can invent something else per user (so we are not confusing people in this example).  Maybe that's better?

Reply:
Yeah, I think that would be better. cc @menghanl who just added go support.

Reply:
Yeah, let's use a different value.

Reply:
ok sounds good!  Fixed!  PTAL thank you!